### DISCUSSION TITLE
MET-11.0.1-rc1 Release Testing

### DISCUSSION THREAD

**Discussion by JohnHalleyGotway** [dtcenter/METplus#2042] [Release Acceptance Testing]
Listed below are the bugfix issues included in the **[MET-11.0.1](https://github.com/dtcenter/MET/milestone/82)** release. This table summarizes the status of external acceptance testing for these issues. The **MET-11.0.1-rc1** tag is the release candidate for the **MET-11.0.1** bugfix release. This discussion tracks the external testing prior to the official **MET-11.0.1** bugfix release.

The **MET-11.0.1-rc1** release is compiled/available for testing on the NOAA Hera machine in the following location:
[TODO: Define location here]

<h2>External Testing Instructions</h2>

For each line in the table, the `External Tester` listed should:
1. Test the **MET-11.0.1-rc1** release to confirm the changes work as expected.
2. **ON THE ISSUE ITSELF**: Document their testing with a comment using the template provided below.
3. **ON THIS DISCUSSION**: Add a comment linking to their issue comment created above.

Copy/paste the following template into your **ISSUE COMMENT**:
```
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date:
Location:
Status (PASS/FAIL):
Description:
```
The `External Tester` is set to `NA` for any issues that do not require external acceptance testing.

See METplus Version 5.0 Support <a href="https://github.com/orgs/dtcenter/projects/69">Project Board</a>

<h2>Testing for specific bugfixes in MET-11.0.1-rc1</h2>

| Status<br>(OPEN/PASS/FAIL)  | MET Issue | External Tester | Link to Acceptance Testing Comment |
|---|---|---|---|
| **PASS** | dtcenter/MET#2218 | @GwenChen-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5081612 |
| OPEN | dtcenter/MET#2380 | @robdarvell | |
| **PASS** | dtcenter/MET#2383 | @perthsb | Partha https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5095754, Julie https://github.com/dtcenter/MET/issues/2383#issuecomment-1440813815 |
| **PASS** | dtcenter/MET#2387 | NA | No external testing required |
| **PASS** | dtcenter/MET#2389 | NA | No external testing required |
|  **PASS** | dtcenter/MET#2390 | @HathewayWill | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5081674 |
| **PASS** | dtcenter/MET#2399 | @malloryprow | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5076771 |
| OPEN | dtcenter/MET#2402 | USAF? | |
| **PASS** | dtcenter/MET#2408 | NA | No external testing required |
| **PASS** | dtcenter/MET#2412 | @GwenChen-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5080149 |
| **PASS*** | dtcenter/MET#2426 | @GwenChen-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5094617 |
| OPEN | dtcenter/MET#2428 | @junmeiban | |
| OPEN | dtcenter/MET#2437 | @craigr3 | |
| **PASS** | dtcenter/MET#2452 | @Ho-ChunHuang-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5079612 |

<h2>Testing for NOAA/EVS v1 use cases on Hera</h2>

NOAA/EMC staff are encouraged to test their existing use cases for EVS v1 on Hera with the **MET-11.0.1-rc1** release. Please document any validation testing you do with a comment **ON THIS DISCUSSION**.

| Status<br>(OPEN/PASS/FAIL)  | Description | External Tester | Link to Acceptance Testing Comment |
|---|---|---|---|
| **PASS** | Subseasonal EVS daily SST | @ShannonShields-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5081910 |
| **PASS** | PB2NC & Point Stat testing | @PerryShafran-NOAA |  https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5088615 |
| **PASS** | CAM precipitation and grid2obs verification use cases | @MarcelCaron-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5092321 |
| **PASS** | Global TC verification | @JiayiPeng-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5123996 |
| **PASS** | EMC/EVS/global_ens and SREF packages | @BinbinZhou-NOAA | https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5126470 |

**Comment by malloryprow** [comment]
@JohnHalleyGotway Hi John, happy to test. Is this on Hera anywhere?

**Reply by JohnHalleyGotway** [reply]
@malloryprow, @GwenChen-NOAA, @craigr3, @junmeiban, @robdarvell, @perthsb, @HathewayWill

Sorry for tagging you on this discussion without providing any context!

The METplus team is brainstorming ideas for how to be a little more formal in seeking input from users in our development process. When we work on new features and bugfixes, we use pull requests to confirm that the changes work as expected, but we really only test in our development environment. And the potential exists that we either didn't fully understand/fix the stated issue or that the new code works differently than expected in the user's environment.

We're meeting with NOAA/EMC folks in the coming weeks to lay out a strategy to track this "external acceptance testing" more closely. And we'll propose using this new [Release Acceptance Testing](https://github.com/dtcenter/METplus/discussions/categories/release-acceptance-testing) GitHub Discussions Category to do so. Since it's *DIFFICULT* to install new s/w versions operationally, we want to confirm that the code changes really do work as expected prior to creating this bugfix release.

So right now, there's nothing to do. Once we coordinate with NOAA/EMC, finalize the process, and actually complete all the fixes for MET-11.0.1, I'll retag you in a comment on this discussion to request your testing.

Thanks!

**Reply by malloryprow** [reply]
Sounds good! Thanks for the info!

**Comment by HathewayWill** [comment]
Thanks

**Reply by HathewayWill** [reply]
Version MET-11.0.1-rc1
Dates: 2/22/203
Location: Ubuntu 22.04, MacOS 13
Status (PASS/FAIL): PASS
Description:  Testing if zlib variable was created and utilized in the code.  Looked through new package and saw the export for LIB_Z.  Tests all successful on multiple systems.

**Comment by malloryprow** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 2/22/2023
Location: Hera, /scratch1/NCEPDEV/global/Mallory.Row/VRFY/MET_issue_2399
Status (PASS/FAIL): PASS
Description: Tested using "regrid_data_plane" to regrid GFS GRIB2 file to grids G172 and G220. Used "plot_data_plane" check the output.
![g220](https://user-images.githubusercontent.com/29127677/220622535-b0e45c15-74be-4e56-80bf-1c4b13556296.png)
![g172](https://user-images.githubusercontent.com/29127677/220622545-50325e5d-73c5-48a5-959f-880ac6acda0d.png)

**Comment by Ho-ChunHuang-NOAA** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 02/22/2023
Location: HERA
Status (PASS/FAIL): PASS
Description: Testing ASCII2NC AirNOW

When using hourly AirNOW file "HourlyData_yyyymmddHH.dat", there is a 11.4% increase on the number of observations when switching from using met/11.0.0 to met/11.0.1-rc1.  This is consistent with the  expectation of bug fix.

input files directory : /scratch2/NCEPDEV/naqfc/Ho-Chun.Huang/noscrub/dcom/20221103/airnow

output file location
/scratch2/NCEPDEV/naqfc/Ho-Chun.Huang/noscrub/test_11.0.0/20221103/test.nc
/scratch2/NCEPDEV/naqfc/Ho-Chun.Huang/noscrub/test_11.0.1_rc1/20221103/test.nc

==========================
export MET_AIRNOW_STATIONS=/scratch2/NCEPDEV/naqfc/Ho-Chun.Huang/noscrub/dcom/20221103/airnow/Monitoring_Site_Locations_V2.dat

met/11.0.0;
ascii2nc HourlyData_2022110300.dat HourlyData_2022110301.dat HourlyData_2022110302.dat HourlyData_2022110303.dat HourlyData_2022110304.dat HourlyData_2022110305.dat HourlyData_2022110306.dat HourlyData_2022110307.dat HourlyData_2022110308.dat HourlyData_2022110309.dat HourlyData_2022110310.dat HourlyData_2022110311.dat HourlyData_2022110312.dat HourlyData_2022110313.dat HourlyData_2022110314.dat HourlyData_2022110315.dat HourlyData_2022110316.dat HourlyData_2022110317.dat HourlyData_2022110318.dat HourlyData_2022110319.dat HourlyData_2022110320.dat HourlyData_2022110321.dat HourlyData_2022110322.dat HourlyData_2022110323.dat /scratch2/NCEPDEV/naqfc/Ho-Chun.Huang/noscrub/test_11.0.0/20221103/test.nc -format airnowhourly

nobs = 162790 ;

met/11.0.1-rc1;
ascii2nc HourlyData_2022110300.dat HourlyData_2022110301.dat HourlyData_2022110302.dat HourlyData_2022110303.dat HourlyData_2022110304.dat HourlyData_2022110305.dat HourlyData_2022110306.dat HourlyData_2022110307.dat HourlyData_2022110308.dat HourlyData_2022110309.dat HourlyData_2022110310.dat HourlyData_2022110311.dat HourlyData_2022110312.dat HourlyData_2022110313.dat HourlyData_2022110314.dat HourlyData_2022110315.dat HourlyData_2022110316.dat HourlyData_2022110317.dat HourlyData_2022110318.dat HourlyData_2022110319.dat HourlyData_2022110320.dat HourlyData_2022110321.dat HourlyData_2022110322.dat HourlyData_2022110323.dat /scratch2/NCEPDEV/naqfc/Ho-Chun.Huang/noscrub/test_11.0.1_rc1/20221103/test.nc -format airnowhourly

nobs = 181378 ;

**Comment by GwenChen-NOAA** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 2/22/2023
Location: Hera
Status (PASS/FAIL): Pass
Description: Fix time interpolation of monthly climatology data between December 15 and January 15 (METplus discussion #2007 and MET issue #2412)

Tested using grid_stat to verify RTOFS SST forecasts with GHRSST data and WOA18 monthly climatology files.

Old log messages from MET 11.0.0:
DEBUG 3: Skipping the 19861215_000000 "t_an(0,0,*,*)" climatology field since the time offset (29376000 seconds) >= the "day_interval" entry (2678400 seconds) from file: /scratch2/NCEPDEV/ovp/Lichuan.Chen/Global/climo/WOA18/woa18_decav_t12_04.nc
...
DEBUG 4: Found matching 19860115_000000 "t_an(0,0,*,*)" climatology field in file "/scratch2/NCEPDEV/ovp/Lichuan.Chen/Global/climo/WOA18/woa18_decav_t01_04.nc".
DEBUG 2: Regridding the 19860115_000000 "t_an(0,0,*,*)" climatology field to the verification grid.
DEBUG 3: For forecast valid at 20230109_000000, found 1 climatology field(s) with valid time(s): 20230115_000000
DEBUG 3: Found 1 climatology fields.
DEBUG 3: Found 1 climatology mean and 0 climatology standard deviation field(s) for forecast sst(0,*,*).

New log messages from MET 11.0.1-rc1:
DEBUG 3: Storing 19861215_000000 "t_an(0,0,*,*)" climatology field with -25 day, 0 hour offset as time 20221215_000000 from file "/scratch2/NCEPDEV/ovp/Lichuan.Chen/Global/climo/WOA18/woa18_decav_t12_04.nc".
DEBUG 2: Regridding 19861215_000000 "t_an(0,0,*,*)" climatology field to the verification grid.
...
DEBUG 3: Storing 19860115_000000 "t_an(0,0,*,*)" climatology field with 6 day, 0 hour offset as time 20230115_000000 from file "/scratch2/NCEPDEV/ovp/Lichuan.Chen/Global/climo/WOA18/woa18_decav_t01_04.nc".
DEBUG 2: Regridding 19860115_000000 "t_an(0,0,*,*)" climatology field to the verification grid.
DEBUG 3: For forecast valid at 20230109_000000, found 2 climatology field(s) with valid time(s): 20221215_000000, 20230115_000000
DEBUG 3: Interpolating climatology fields at 20221215_000000 and 20230115_000000 to 20230109_000000 using the DW_MEAN interpolation method.
DEBUG 3: Found 1 climatology fields.
DEBUG 3: Found 1 climatology mean and 0 climatology standard deviation field(s) for forecast sst(0,*,*).

**Comment by GwenChen-NOAA** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 2/22/2023
Location: Hera
Status (PASS/FAIL): Pass
Description: Fix the MET CF-Compliant NetCDF library code to Polar Stereographic data from NSIDC Sea Ice Edge NetCDF files (METplus discussion #1718 and MET issue #2218)

Tested using plot_data_plane to read NSIDC and OSISAF sea ice data.  Both are in polar stereographic grid.

![NSIDC0081_SEAICE_PS_N25km_20220723_v2 0](https://user-images.githubusercontent.com/95313292/220751558-64aa8f4b-f7e4-485c-a9d7-8721c918f3d2.png)
![NSIDC0081_SEAICE_PS_S25km_20220723_v2 0](https://user-images.githubusercontent.com/95313292/220751630-4f68682d-437b-427c-ab9f-368c9625579e.png)
![ice_conc_nh_polstere-100_multi_202302211200](https://user-images.githubusercontent.com/95313292/220751665-987e408e-7920-4d04-be49-56088319285c.png)
![ice_conc_sh_polstere-100_multi_202302211200](https://user-images.githubusercontent.com/95313292/220751729-51c14308-3ba3-42ae-8e25-e9da9f3c9255.png)

**Comment by HathewayWill** [comment]
Version MET-11.0.1-rc1
Dates: 2/22/203
Location: Ubuntu 22.04, MacOS 13
Status (PASS/FAIL): PASS
Description: Testing if zlib variable was created and utilized in the code. Looked through new package and saw the export for LIB_Z. Tests all successful on multiple systems.

**Comment by ShannonShields-NOAA** [comment]
**Release Acceptance Testing Summary**

Version: MET-11.0.1-rc1
Date: 2/22/2023
Location: Hera
Status (PASS/FAIL): PASS
Description: Ran basic test for subseasonal EVS daily SST (with CFS model data and GHRSST OSPO obs data) using GenEnsProd, GridStat, and StatAnalysis tools. Successfully generated nc pairs and stat files.
Tested looping capability of StatAnalysis associated with [#2027](https://github.com/dtcenter/METplus/pull/2027) for METplus-5.0.1-rc1.

**Comment by jprestop** [comment]
Here is a [link](https://github.com/dtcenter/MET/issues/2383#issuecomment-1440813815) to my comment on the issue [Bugfix: Fix ASCII2NC to not compute AOD 550 if other inputs are negative values #2383](https://github.com/dtcenter/MET/issues/2383#top) for Partha ([@perthsb](https://github.com/perthsb)).

**Comment by PerryShafran-NOAA** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 2/22/2023
Location: Hera
Status (PASS/FAIL): PASS
Description: PB2NC & Point Stat testing

Ran a comparison of PB2NC and Point Stat, comparing older versions METplus 5.0.0 and MET 11.0.0 vs the new candidates METplus 5.0.1-rc1 and MET 11.0.1-rc1.  The resulting stat files were identical.

**Comment by malloryprow** [comment]
Not sure if this is okay to put here or not...

Doing some testing on Hera and noticed some differences with the NBRCNT stats. Thought I'd post here to have a discussion before roping in the METplus team
V11.0.0 on WCOSS2

> V11.0.0 gfs G240 960000  20230215_120000 20230215_120000 240000 20230215_120000 20230215_120000 APCP  mm    A24    APCP  mm    A24    24hrCCPA CONUS    NBRHD_SQUARE 1     >=0.254  >=0.254  NA    0.05 NBRCNT  453300  **0.21389**     NA       NA    **0.82862**       NA   NA **0.99556**  NA  NA **0.79731**  NA  NA  0.65347  NA  NA  **0.59461**  NA  NA

V11.0.1 on Hera

> V11.0.1 gfs G240 960000  20230215_120000 20230215_120000 240000 20230215_120000 20230215_120000 APCP  mm    A24    APCP  mm    A24    24hrCCPA CONUS    NBRHD_SQUARE 1     >=0.254  >=0.254  NA    0.05 NBRCNT  453300  **0.21406**     NA       NA    **0.82846**       NA   NA **0.99553**  NA  NA **0.79721**  NA  NA  0.65347  NA  NA  **0.59442**  NA  NA

**Reply by malloryprow** [reply]
Must be a Hera vs. WCOSS2 thing...reran on Hera with V11.0.0

> V11.0.0 gfs   G240 960000    20230215_120000 20230215_120000 240000   20230215_120000 20230215_120000 APCP     mm         A24        APCP    mm        A24        24hrCCPA CONUS         NBRHD_SQUARE 1           >=0.254     >=0.254    NA         0.05  NBRCNT    453300     0.21406          NA              NA         0.82846              NA       NA   0.99553    NA    NA 0.79721    NA    NA    0.65347    NA    NA    0.59442    NA    NA

**Reply by JohnHalleyGotway** [reply]
Aha, thanks for following up Mallory. I just logged onto Hera to run exactly that same test... NBRCNT output on Hera for v11.0.0 vs v11.0.1-rc1. Glad to hear they are identical. We did discuss this today in the MET development meeting and were theorizing that it probably was a platform/compiler difference.

Differences in the 4th or 5th decimal place are certainly not unexpected on different platforms. But differences in the 3rd decimal place are much less comfortable.

In years past, we did run the MET unit tests on different platforms (cheyenne, jet, RAL Linux machines) prior to releases and noted similar differences in the output. But we haven't prioritized cross-platform testing after adopting automated testing via GitHub actions. We do have hopes/plans to extend GHA to test out different compiler families/versions, which we could use to better quantify the scale of these types of diffs.

But I'll leave it to EMC to discuss how big of a priority that type of testing would be.

**Comment by MarcelCaron-NOAA** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 2023-02-23
Location: Hera
Status (PASS/FAIL): PASS
Description: Tested CAM precipitation and grid2obs verification use cases.  MET tools: PCPCombine, GridStat, StatAnalysis, GenVxMask, PB2NC, PointStat.  All test jobs ran successfully.  Nearly identical stat files in test comparisons with MET-10.1.2 WCOSS2 runs; differences in NBRCNT begin at the 3rd-5th decimal places in the samples I looked at.  Example comparison:
**MET-10.1.2:**
`V10.1.2 namnest NA 120000 20230219_120000 20230219_120000 000000 20230219_120000 20230219_120000 APCP_03 kg/m^2 A3 APCP_03 kg/m^2 A3 CCPA CONUS NBRHD_SQUARE 1 >=0.254 >=0.254 NA 0.05 NBRCNT 453300 0.031544 NA NA 0.32555 NA NA 0.7487 NA NA 0.50726 NA NA 0.03225 NA NA 0.01452 NA NA`
**MET-11.0.1-rc1:**
`V11.0.1 namnest NA 120000 20230219_120000 20230219_120000 000000 20230219_120000 20230219_120000 APCP_03 kg/m^2 A3 APCP_03 kg/m^2 A3 CCPA CONUS NBRHD_SQUARE 1 >=0.254 >=0.254 NA 0.05 NBRCNT 453300 0.031529 NA NA 0.32699 NA NA 0.75133 NA NA 0.5073 NA NA 0.03225 NA NA 0.014597 NA NA`

**Comment by GwenChen-NOAA** [comment]
<h3>Release Acceptance Testing Summary</h3>

Version: MET-11.0.1-rc1
Date: 2/24/2023
Location: Hera
Status (PASS/FAIL): Pass
Description: Fix ASCII2NC to handle missing NDBC buoy location information (MET issue #2426)

Tested using ASCII2NC with 20230223 NDBC data on WCOSS2 at /lfs/h1/ops/dev/dcom/20230223/validation_data/marine/buoy. NDBC data were copied to Hera for testing.

Old log messages from MET 11.0.0:
ERROR  : No location information found for station 44084 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/44084.txt
ERROR  : No location information found for station 46275 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/46275.txt
ERROR  : No location information found for station CSXA2 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/CSXA2.txt
ERROR  : No location information found for station DILA1 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/DILA1.txt
ERROR  : No location information found for station KBMG1 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/KBMG1.txt
ERROR  : No location information found for station KCXA2 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/KCXA2.txt
ERROR  : No location information found for station KOZA2 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/KOZA2.txt
ERROR  : No location information found for station OWMO1 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/OWMO1.txt
ERROR  : No location information found for station PAUA2 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/PAUA2.txt
ERROR  : No location information found for station SGXA2 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/SGXA2.txt
ERROR  : No location information found for station WCRP1 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/WCRP1.txt
ERROR  : No location information found for station WGXA2 do not process file /scratch2/NCEPDEV/ovp/Lichuan.Chen/NDBC/20230223/WGXA2.txt

New log messages from MET 11.0.1-rc1:
DEBUG 1: Number of NDBC skipped files due to no lookup 0

**Comment by ghost** [comment]
Version: MET-11.0.1-rc1
Date: 2023-02-24
Location: Hera
Status (PASS/FAIL): PASS
Description : Tested Metplus asciitonc tool to convert multiple AERONET station data in a file to successfully create a netcdf file with all stations  input AOD interpolated to 550nm. No errors to convert them to netcdf. 
Logfile for asciitonc : /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/Scripts/output.txt
Logfile for pointstat : /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/Aeronet_Download/logs/metplus.log.20230223182734
Dump txt file for asciitonc : /scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/Scripts/All_20221019.lev15.dump.txt
grep AOD scratch1/NCEPDEV/stmp2/Partha.Bhattacharjee/METplus/Scripts/Scripts/All_20221019.lev15.dump.txt | grep ' 550.00 ' | wc -l
12783

Thanks

**Comment by JiayiPeng-NOAA** [comment]
Hi John,
I have tested "met/11.0.1-rc1 and metplus/5.0.1-rc1" for global TC verifications on HERA.
It works with this bug-fixed version.
Please see the plots and stats on: https://ftp.emc.ncep.noaa.gov/gc_wmb/jpeng/tc_verify_onHera_230226/
Thanks,
Jiayi

**Comment by JohnHalleyGotway** [comment]
@craigr3, @junmeiban, @robdarvell.

The METplus team is planning to create the MET-11.0.1 bugfix release tomorrow, 2/28/23. Your names are [flagged above](https://github.com/dtcenter/METplus/discussions/2042#discussion-4841293) in this discussion as the requestors of 3 of the fixes for this release.

If you'd like to do any specific testing of these issues in the [MET-11.0.1-rc1 release](https://github.com/dtcenter/MET/releases/tag/v11.0.1-rc1) or provide any other feedback about these fixes, please do so by 2/28/23. 

Rest assured that we have tested these fixes internally. This **Release Acceptance Testing** discussion gives requestors an opportunity to provide additional feedback prior to us creating the release.

Please provide any feedback you may have as a comment on this discussion or as a comment on the GitHub bugfix issue itself.

Thanks!

**Comment by BinbinZhou-NOAA** [comment]
**Release Acceptance Testing Summary**
Version: MET/11.0.1-rc1, METplus/5.0.1-rc1
Date: 2023-02-27
Location: Hera
Status (PASS/FAIL): PASS
Description: Tested EMC/EVS/global_ens and SREF  packages for  MET tools: PCPCombine, GenEnsProd, EnsembleStat, PointStat and GridStat.  All of the testing jobs ran successfully. The output stat files were compared with the output stat files from  MET/11.0.0 and METplus/5.0.0 on  WCOSS2 runs,  showing that they are exactly the same.

**Comment by BinbinZhou-NOAA** [comment]
I just noticed that there is extra column called "EC_VALUE" in the CTC line type in MET/11.0.0. This does not impact the verification, but may cause failure for out plotting. In the METplus/5.0.0 user guide, there is a new variable "HSS_EC_VALUE".  Is this the specific variable to not print  the EC_VALUE in the CTC line type data?

**Reply by JohnHalleyGotway** [reply]
Binbin, this was added to MET version 11.0.0. The EC_VALUE is the "expected correct" value and is used in the computation of the HSS_EC statistics reported in the [CTS line type](https://met.readthedocs.io/en/latest/Users_Guide/point-stat.html#id9).

There is no way to disable its output. This was added at the request of NOAA CPC which needs the ability to control the expected correct value when computing Heidke Skill Score. The default value is 0.5. It is configurable via the [HSS_EC_VALUE](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html?highlight=hss_ec_value#hss-ec-value) config file option.

It is included in the CTC line type so that when those lines are aggregated via METviewer or Stat-Analysis, they are able to compute the correct HSS_EC statistic.

**Comment by BinbinZhou-NOAA** [comment]
John,

  OK, no problem for the EC_VALUE being added to CTC output.  After
modifying my plotting scripts,
the  plotting  works well for the new CTC line type output from  MET
version 11.0.0.
Thanks!

Binbin

On Tue, Mar 7, 2023 at 11:31 AM John Halley Gotway ***@***.***>
wrote:

> Binbin, this was added to MET version 11.0.0. The EC_VALUE is the
> "expected correct" value and is used in the computation of the HSS_EC
> statistics reported in the CTS line type
> <https://met.readthedocs.io/en/latest/Users_Guide/point-stat.html#id9>.
>
> There is no way to disable its output. This was added at the request of
> NOAA CPC which needs the ability to control the expected correct value when
> computing Heidke Skill Score. The default value is 0.5. It is configurable
> via the HSS_EC_VALUE
> <https://met.readthedocs.io/en/latest/Users_Guide/config_options.html?highlight=hss_ec_value#hss-ec-value>
> config file option.
>
> It is included in the CTC line type so that when those lines are
> aggregated via METviewer or Stat-Analysis, they are able to compute the
> correct HSS_EC statistic.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dtcenter/METplus/discussions/2042#discussioncomment-5230935>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJ2AOLUMBA6QZTJI23W25PGRANCNFSM6AAAAAAUWY5GZY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
